src/format_rules.ml

```diff
@@ -77,7 +76,14 @@ let gen_rules sctx (config : Dune_file.Auto_format.t) ~dir =
           ; Target output
           ]
         in
-        Some (Lazy.force ocamlformat_deps >>> Build.run ~dir exe args)
+        Some
```

Comment:
You should stick to a consistent formatting of `Some`. For example, change this to how you formatted it the second time in this file.